object_storage_api/stores/attachment.py

```diff
@@ -88,3 +88,22 @@ def delete(self, object_key: str) -> None:
             Bucket=object_storage_config.bucket_name.get_secret_value(),
             Key=object_key,
         )
+
+    def delete_many(self, object_keys: list[str]) -> None:
+        """
+        Deletes given attachments from object storage by object keys.
+
+        It does this in batches due to the `delete_objects` request only allowing a list of up to 1000 keys.
+
+        :param object_keys: Keys of the attachments to delete.
+        """
+        logger.info("Deleting attachment files with object keys: %s from the object store", object_keys)
+
+        batch_size = 1000
+        # Loop through the list of object keys in steps of `batch_size`
+        for i in range(0, len(object_keys), batch_size):
+            batch = object_keys[i : i + batch_size]
+            s3_client.delete_objects(
+                Bucket=object_storage_config.bucket_name.get_secret_value(),
+                Delete={"Objects": [{"Key": key for key in batch}]},
```

Comment:
```suggestion
                Delete={"Objects": [{"Key": key } for key in batch]},
```
The current implementation works, but I'm not sure why it does?

The docs for the function requires the parameter to be structured as so:
![image](https://github.com/user-attachments/assets/f417bafd-4bb2-4cd9-b907-e33cefa964b0)

If I modify the code to compare the 2 outputs of both options this is the result.
![image](https://github.com/user-attachments/assets/e9184f7b-50a1-4f94-8328-4de134eda2a6)
![image](https://github.com/user-attachments/assets/c22f1161-e205-4dde-9cfb-cff9baf02f16)

Again both implementation work, but I don't understand why yours works?